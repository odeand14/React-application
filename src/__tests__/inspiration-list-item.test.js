import React from 'react';
import Item from '../inspiration-list-item';
import renderer from 'react-test-renderer';

it('renders a snapshot', () => {
    const tree = renderer
        .create(<Item/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});

