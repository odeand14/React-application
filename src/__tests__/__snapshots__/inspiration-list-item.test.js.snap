// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<tbody>
  <tr>
    <td
      className="lead"
    />
    <td
      className="lead"
    />
    <td
      className="lead"
    />
  </tr>
</tbody>
`;
